src/sentinel.c

```diff
@@ -1523,14 +1523,15 @@ int sentinelResetMasterAndChangeAddress(sentinelRedisInstance *master, char *ip,
     newaddr = createSentinelAddr(ip,port);
     if (newaddr == NULL) return C_ERR;
 
+    slaves = zmalloc(sizeof(sentinelAddr*)*(dictSize(master->slaves) + 1));
```

Comment:
Would you mind documenting why it's +1, since that isn't that obvious?

Reply:
Hi, @madolson, It is because that next of that part, there is a code add 1 more into slave,  when 
 if (!sentinelAddrIsEqual(newaddr,master->addr)). Thanks.

Reply:
:D I meant put a comment in the code.

Reply:
@madolson applied. Thanks.

Reply:
Thanks!